styles.css

```diff
@@ -0,0 +1,463 @@
+  position: relative;
+  background-image: image("images/header-image.jpg");
+  overflow: hidden;
+}
+
+.header-img {
+  width: 100%;
+  margin-right: 0;
+}
+h1 {
+  color: #44BE2F;
+  padding-top: 26%;
+  padding-left: 3rem;
+  font-weight: 500;
+  font-style: italic;
+}
+
+.header-logo {
+  text-align: left;
+  position: absolute;
+  top: 2rem;
+  left: 5rem;
+}
+
+.header-checkout {
+  text-align: left;
+  position: absolute;
+  top: 10px;
+  right: 2%;
+  font-family: monospace;
+  font-weight: 700;
+  font-size: 1em;
+  text-decoration: none;
+  color: #fff;
+}
+
+.header-checkout a:link {
+  text-decoration: none;
+  padding: 4px 8px 4px 8px;
+  background-color: #9bc344;
+  border-radius: 1px;
+  border: none;
+  width: 240px;
+  color: #fff;
+}
+
+.header-checkout a:visited {
+  color: #fff;
+  background-color: #9bc344;
+}
+
+.header-checkout a:hover {
+  color: #fff;
+  background-color: #6a872c;
+}
+
+.header-checkout a:active {
+  color: #fff;
+  background-color: #103861;
+}
+
+
+.header-checkout img {
+  padding-right: 0;
+}
+
+.header-checkout a:link {
+  text-decoration: none;
+  padding: .5em 1em .5em 1em;
+  background-color: #9bc344;
+  border-radius: .5em;
+  border: none;
+  width: 20em;
+  color: #fff;
+}
+
+.header-checkout a:visited {
+  color: #fff;
+  background-color: #9bc344;
+}
+
+.header-checkout a:hover {
+  color: #fff;
+  background-color: #6a872c;
+}
+
+.header-checkout a:active {
+  color: #fff;
+  background-color: #103861;
+}
+
+nav {
+  background-color: #D3D5D3;
+  margin-top: 0px;
+  padding-top: .5em;
+  padding-bottom: .5em;
+  font-family: monospace;
+  padding-left: 2em;
+  font-size: 1.5em;
+
+}
+nav a {
+  text-decoration: none;
+  color: #103861;
+}
+
+nav a:hover {
+  text-decoration: underline;
+}
+.container {
+  margin-left: 5%;
+}
+h2 {
+  font-family: serif;
+  font-weight: 700;
+  font-size: 2em;
+  margin-top: 2em;
+  color: #103861;
+}
+
+h3 {
+  font-family: sans-serif;
+  font-weight: 700;
+  font-size: 1em;
+  color: #103861;
+  padding-left: 1.5rem;;
+}
+
+h4 {
+  font-family: serif;
+  font-weight: 700;
+  font-size: 1em;
+  color: #103861;
+}
+
+h5 {
+  font-family: serif;
+  font-weight: 700;
+  font-size: 1em;
+  color: #103861;
+}
+
+p {
+  font-family: sans-serif;
+  font-weight: 300;
+  font-size: 1em;
+  padding-left: 2em;
+}
+
+.home-container {
+  margin: 1em 2em 2em 0em;
+  max-width: 90%;
+}
+
+.home-container img {
+  float: left;
+  padding-right: 1em;
+}
+
+.home-container h2 p {
+  float: right;
+  padding: 1em 4em 4em 0em;
+}
+
+.dog-profiles {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  clear: both;
+  max-width: 90%;
+  padding-top: 2em;
+}
+
+.dog-profiles section {
+  border: .1em solid #ccc;
+  width: 10em;
+  margin: 1em 3em 2em 0em;
+}
+
+.dog-profiles a:link {
+  font-family: sans-serif;
+  font-weight: 700;
+  font-size: .7em;
+  margin-top: 20px;
+  margin-bottom: 0px;
+  padding: 4px 70px 4px 62px;
```

Comment:
You should not be using px padding to set the size. 
You could instead try setting display to block and width to 100%